src/tag/types.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {OverrideT} from '../helpers/overrides';
+
+export type OverridesT = {
+  Root?: OverrideT<*>,
+  Action?: OverrideT<*>,
+};
+
+export type DefaultPropsT = {
+  overrides: OverridesT,
+  disabled: boolean,
+  isFocused: boolean,
+  isHovered: boolean,
+  isActive: boolean,
+  color: string,
+  onActionClick: (
+    e: SyntheticEvent<HTMLInputElement>,
+    children: React$Node,
+  ) => void,
+};
```

Comment:
@abmai Just had a PR where he removed a `DefaultPropsT` object to get flow correctly identify required props #236 

Should we remove the DefaultPropsT here to stay consistent? (I'm guessing there may be other existing components we'll need to change in the future too)

Reply:
ok, still works